_inc/lib/tracks/class.tracks-client.php

```diff
@@ -182,8 +182,12 @@ static function get_connected_user_tracks_identity() {
 		if ( ! $user_data = Jetpack::get_connected_user_data() ) {
 			return false;
 		}
+		if ( ! $blog_id = Jetpack_Options::get_option( 'id' ) ) {
```

Comment:
Do we really want to make the blog ID required? Wont this break for cases where the site is not connected currently? TBH I expected the blog ID to be optional here.

Reply:
good point.